authz/sdk_server_interceptors_test.go

```diff
@@ -19,19 +19,43 @@
 package authz_test
 
 import (
+	"io/ioutil"
+	"os"
+	"path"
 	"testing"
+	"time"
 
 	"google.golang.org/grpc/authz"
 )
 
+func createTmpPolicyFile(t *testing.T, dirSuffix string, policy []byte) string {
+	t.Helper()
+
+	// Create a temp directory. Passing an empty string for the first argument
+	// uses the system temp directory.
+	dir, err := ioutil.TempDir("", dirSuffix)
+	if err != nil {
+		t.Fatalf("ioutil.TempDir() failed: %v", err)
+	}
+	t.Logf("Using tmpdir: %s", dir)
+	// Write policy into file.
+	filename := path.Join(dir, "policy.json")
+	if err := ioutil.WriteFile(filename, policy, os.ModePerm); err != nil {
+		t.Fatalf("ioutil.WriteFile(%q) failed: %v", filename, err)
+	}
+	t.Logf("Wrote file at: %s", filename)
+	t.Logf("%s", string(policy))
```

Comment:
`t.Log(string(policy))`

Reply:
Appended the policy log to previous log statement.